### ISSUE TITLE
Unable to find image 'whisk/runtimes/action-nodejs-v10:prod-build-16714_1.15.1'

### ISSUE THREAD

**Issue by playground** [apache/openwhisk-wskdebug#36]
When try to run the example 
`wskdebug` wskdebug-examples/webaction webaction.js -l `

throws these errors
`+------------------------------------------------------------------------------------------+
| Docker image must be downloaded: whisk/runtimes/action-nodejs-v10:prod-build-16714_1.15.1
|                                                                                          |
| Note: If you debug in VS Code and it fails with "Cannot connect to runtime process"      |
| due to a timeout, run this command once:                                                 |
|                                                                                          |
|     docker pull whisk/runtimes/action-nodejs-v10:prod-build-16714_1.15.1
|                                                                                          |
| Alternatively set a higher 'timeout' in the launch configuration, such as 60000 (1 min). |
+------------------------------------------------------------------------------------------+

Unable to find image 'whisk/runtimes/action-nodejs-v10:prod-build-16714_1.15.1' locally
docker: Error response from daemon: pull access denied for whisk/runtimes/action-nodejs-v10, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.

Error: Command failed: docker run -d --name wskdebug-webaction-1586472788047 --rm -m 268435456 -p 8080 -p 9229:9229 -v "/Users/jeff/git_repo/sandbox/wu/playbox/openwhisk-wskdebug/examples/nodejs:/code" whisk/runtimes/action-nodejs-v10:prod-build-16714_1.15.1 node --expose-gc --inspect=0.0.0.0:9229 app.js `

**Comment by alexkli** [comment]
**tldr;** Try overwriting the image and use the standard openwhisk image for nodejs 10:

```
wskdebug --image openwhisk/action-nodejs-v10:latest wskdebug-examples/webaction webaction.js -l
```

**Explanation:**

The image `whisk/runtimes/action-nodejs-v10:prod-build-16714_1.15.1` is what your openwhisk system uses for `nodejs` kinds. `wskdebug` needs it to run the action locally in the container. Looks like this image is not publicly available. Maybe it's in a private docker registry and you need to set up your docker with that registry.
